### PR TITLE
Commercial ops

### PR DESCRIPTION
**Pull request by LJGarvie**
Commercial ops principles added

### CODE REVIEW

File: playbook.md [hunk 1]
```diff
@@ -820,6 +820,44 @@ Designers at dxw follow these principles:
 
    There is always room for improvement. Design can always be made better. There is a difference between ‘done’ and ‘finished’.
 
+### Commercial Operations
+
+We are the team that enables and encourages dxw to achieve its missions(s) and uphold its values.
```
Comment (CristinaRO) [inline playbook.md]: "missions(s)" - possibly "mission(s)" instead?

File: playbook.md [hunk 2]
```diff
@@ -820,6 +820,44 @@ Designers at dxw follow these principles:
...
+
+1. Support each other to support the team.
+
+1. Commercial accuracy and responsibility.
+
+1. We are balanced, organised and resilient.
+
+1. People over process, to maintain our unique, small-company, culture.
+
+1. We are proud of what we do and our confident enough to talk about it.
```
Comment (CristinaRO) [inline playbook.md]: "our confident" --> "are confident"

File: playbook.md [hunk 3]
```diff
@@ -820,6 +820,44 @@ Designers at dxw follow these principles:
 
    There is always room for improvement. Design can always be made better. There is a difference between ‘done’ and ‘finished’.
 
+### Commercial Operations
+
+We are the team that enables and encourages dxw to achieve its mission(s) and uphold its values.
+
+#### Commercial Operations Team Principles
+
+### Things we do:
```
Comment (CristinaRO) [inline playbook.md]: Another structure-related comment: team names are level 3 headings (`###`), the "principles" heading is level 4, so I think "Things we do" and "Behaviours" should be level 5. (They're subordinate to the principles.)

**Comment by lozette** [comment]
Resolved the conflicts!

**Comment by CristinaRO** [comment]
@lozette I think some editor files have sneaked in, files starting with `.idea`.

**Comment by lozette** [comment]
> @lozette I think some editor files have sneaked in, files starting with `.idea`.

Yikes! Thank you!

**Pull request merged**